trunk/bior_annotate.sh

```diff
@@ -413,98 +400,156 @@ NUM_CATALOGS_TO_DRILL=`awk 'END{print NR }' $drills`
 ###
 ##################################################################################
 
-CWD_VCF=`basename $VCF`
-#Check for compressed version
-#Remove IDs from VCFs b/c bior uses them incorrectly
-if [[ "$CWD_VCF" == *gz ]] ;
-then
-	zcat $VCF|\
-	$PERL $VCF_SPLIT|\
-	grep -v 'NON_REF'|\
-	 $PERL -pne 's/[ |\t]$//g'|\
-	$PERL -ne 'if($_!~/^#/){$_=~s/ //g;@line=split("\t",$_);$rsID=".";print join("\t",@line[0..1],$rsID,@line[3..@line-1])}else{print}'|\
-	$VT/vt normalize $VT_OPTIONS -r $REF_GENOME - >${CWD_VCF%%.*}.vcf
-	CWD_VCF=${CWD_VCF%%.*}.vcf
-else
-	cat $VCF|\
-	$PERL $VCF_SPLIT |\
-	grep -v 'NON_REF'|\
-	$PERL -pne 's/[ |\t]$//g'|\
-	$PERL -ne 'if($_!~/^#/){$_=~s/ //g;@line=split("\t",$_);$rsID=".";print join("\t",@line[0..1],$rsID,@line[3..@line-1])}else{print}' |\
-	$VT/vt normalize $VT_OPTIONS -r $REF_GENOME - > $CWD_VCF
-fi
+# Get the output filename without the .gz extension (if it was used)
+VCF_NO_SAMPLES=`basename $VCF | sed 's/\.gz$//g'`
+LINE_NUMS_AND_SAMPLES_FILE=lineNumsAndSamples.tsv.gz
 
-if [ "$CLINICAL" == "TRUE" ]
-then
-	#Split PEDIGREE and CWD_VCF to only contain the 3 target samples
+# GROOVY_HOME should be set from the tool_info.txt file
+# Ex: export GROOVY_HOME=/data5/bsi/bictools/src/groovy/2.4.7/
+if [ -z "$GROOVY_HOME" ] || [ ! -d "$GROOVY_HOME" ]; then
+  echo "ERROR: Could not find GROOVY_HOME path or GROOVY_HOME was not an existing directory: $GROOVY_HOME"
+  exit 1
+fi
+export PATH=$GROOVY_HOME/bin:$PATH
+
+#----------------------------------------------------------
+# Remove the sample and format columns  (NOTE: THIS *MUST* BE DONE BEFORE THE SPLIT OR THE LINES WON'T MATCH THE ORIGINAL VCF!)
+# Split the VCF by ALTS (taking into account other fields related to ALTS)
+# Remove the lines containing "NON_REF" (???)
+# Remove space or tabs at end of lines
+# If line is NOT header, split it by tab (and????)
+# Run VT-Normalize to trim alt and ref strings as necessary
+# Remove IDs from VCFs b/c bior uses them incorrectly
+# NOTE:  VCF_split.pl adds columns: FORMAT, SAMPLE as 9th and 10th cols
+#----------------------------------------------------------
+zcat -f $VCF|\
```

Comment:
@michaelmeiners What if the file is not compressed?  Is that what the `-f` option is for>